daft/unity_catalog/unity_catalog.py

```diff
@@ -90,12 +90,15 @@ def _paginated_list_tables(client: unitycatalog.Unitycatalog, page_token: str |
 
         return self._paginate_to_completion(_paginated_list_tables)
 
-    def load_table(self, table_name: str, new_table_storage_path: str | None = None) -> UnityCatalogTable:
+    def load_table(
+        self, table_name: str, new_table_storage_path: str | None = None, intent: str = "READ_WRITE"
+    ) -> UnityCatalogTable:
```

Comment:
`intent` seems a little weird to me. Maybe we should stick with something closer to python's `open()` function https://docs.python.org/3/library/functions.html#open

So the parameter could be `mode`, and we can do `Literal["r"|"rw"]` as the modes.

Reply:
Sure.  I didn't use Literal because CONTRIBUTING.md mentions Python 3.7 while Literal was introduced in 3.8. I see now that Literal does appear elsewhere in the code, and the project config actually asks for 3.9 so I will go ahead and use it, and update the documentation.

Reply:
Ah good catch, and thanks! Also I just took a look at UC's api and realized they use "READ_WRITE" so we can stick with what you have for the parameter values! Maybe changing `intent` to `operation` could still be good